pkg/ccl/streamingccl/streamproducer/stream_lifetime.go

```diff
@@ -220,14 +220,12 @@ func updateReplicationStreamProgress(
 				return nil
 			}
 
-			// TODO(casper): Error out when the protected timestamp moves backward as the ingestion
-			// processors may consume kv changes that are not protected. We are fine for now
-			// for the sake of long GC window.
-			// Now this can happen because the frontier processor moves forward the protected timestamp
-			// in the source cluster through heartbeats before it reports the new frontier to the
-			// ingestion job resumer which later updates the job high watermark. When we retry another
-			// ingestion using the previous ingestion high watermark, it can fall behind the
-			// source cluster protected timestamp.
+			// Error out when the protected timestamp moves backward as the ingestion processors
+			// may consume kv changes that are not protected. This can happen because the frontier
+			// processor moves forward the protected timestamp in the source cluster through
+			// heartbeats before it reports the new frontier to the ingestion job resumer which
+			// later updates the job high watermark. When we retry another ingestion using the
+			// previous ingestion high watermark, it can fall behind the source cluster protected timestamp.
```

Comment:
i was not clear. I was suggesting that we remove this comment all together. I think you could add a comment along the lines of: "we only update the PTS if it pushes the pts forward. we may receive heartbeat timestamps less than the current protected time after the user issues a cutover time, in which case, we'll receive heartbeats with the cutover time"

Reply:
notice that we _dont_ error out here